src/server/scripts/Commands/cs_ticket.cpp

```diff
@@ -192,7 +192,10 @@ class ticket_commandscript : public CommandScript
         ticket->SaveToDB(trans);
         sTicketMgr->UpdateLastChange();
 
-        std::string msg = ticket->FormatMessageString(*handler, nullptr, ticket->GetAssignedToName().c_str(), nullptr, nullptr);
+        std::string msg = [&] {
```

Comment:
There's no reason to use a lambda here.

Reply:
why? Explain your opinion plz with args

Reply:
Why would you in the first place? If you want to have an enclosed scope then just make an enclosed scope. By doing this you're creating unnecessary clutter and a lambda that has to catch every local by ref.

Reply:
ok its good arg i agree, but as you can see in main post i just port it from TC and not me original author.
Proof: https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/scripts/Commands/cs_ticket.cpp#L207

Reply:
https://github.com/TrinityCore/TrinityCore/commit/29826a92ef4cf7896e3be69757b015d47f3db006#diff-98d8f29d565bf98320917c18fbd1a9b8

This seems to be the commit. That commit is bad as well. Just because TC has it doesn't mean it should be ported like that. That original commit is bad code.

Reply:
rewrite it as soon as i can